x-pack/performance/configs/cloud_security_posture_config.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FtrConfigProviderContext } from '@kbn/test';
+
+// eslint-disable-next-line import/no-default-export
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const xpackFunctionalConfig = await readConfigFile(
+    // eslint-disable-next-line @kbn/imports/no_boundary_crossing
+    require.resolve('../../test/functional/config.base.js')
+  );
```

Comment:
Is there any reason why we can't use the existing config for the FTR functional tests:
`x-pack/test/cloud_security_posture_functional/config.ts` ?

The idea of the journeys is to get read of config files in favour of dynamic configuration, that extends the default config file.

Plus it might be easier to have a single source for servers configurations (if you add a new argument, it will apply to all the tests)

Reply:
I don't use the FTR functional test configuration to prevent dependencies between the tests. 
There may be a need to modify the configuration on one without affecting the other.